src/main/java/seedu/address/storage/JsonAdaptedLesson.java

```diff
@@ -0,0 +1,110 @@
+package seedu.address.storage;
+
+import java.time.LocalTime;
+import java.time.format.DateTimeParseException;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.lesson.Day;
+import seedu.address.model.lesson.Lesson;
+import seedu.address.model.lesson.NumberOfOccurrences;
+import seedu.address.model.lesson.Venue;
+import seedu.address.model.lesson.exceptions.InvalidDayException;
+
+/**
+ * Jackson-friendly version of {@link Lesson}.
+ */
+public class JsonAdaptedLesson {
+
+    public static final String MISSING_FIELD_MESSAGE_FORMAT = "Lesson's %s field is missing!";
+    public static final String INVALID_FIELD_MESSAGE_FORMAT = "Lesson's %s field is invalid!";
+    public static final String START_TIME_FIELD = "start time";
+    public static final String END_TIME_FIELD = "end time";
+
+    private final String startTime;
+    private final String endTime;
+    private final String day;
+    private final int numberOfOccurrences;
+    private final String venue;
+
+    /**
+     * Constructs a {@code JsonAdaptedLesson} with the given lesson details.
+     */
+    @JsonCreator
+    public JsonAdaptedLesson(@JsonProperty("startTime") String startTime,
+                             @JsonProperty("endTime") String endTime,
+                             @JsonProperty("day") String day,
+                             @JsonProperty("numberOfOccurrences") int numberOfOccurrences,
+                             @JsonProperty("venue") String venue) {
+        this.startTime = startTime;
+        this.endTime = endTime;
+        this.day = day;
+        this.numberOfOccurrences = numberOfOccurrences;
+        this.venue = venue;
+    }
+
+    /**
+     * Converts a given {@code Lesson} into this class for Jackson use.
+     */
```

Comment:
Perhaps it is a good idea to keep the javadoc comment consistent with JsonAdaptedModuleClass..
```suggestion
    /**
     * Converts a given {@code Lesson} into a {@code JsonAdaptedLesson} for Jackson use.
     */